source/developers/content-modeling.rst

```diff
@@ -551,47 +551,106 @@ Every Form Control has a Variable Name property.  The Variable Name is used by t
++----------------------+------------------------------------------------------------------+
+|| width               || Used by the deployer                                            |
++----------------------+------------------------------------------------------------------+
+|| height              || Used by the deployer                                            |
++----------------------+------------------------------------------------------------------+
+|| contentLength       || Name of field for file size.  Used by the deployer              |
++----------------------+------------------------------------------------------------------+
+|| lastEditedOn        || Name of field for last edit date. Used by the deployer          |
++----------------------+------------------------------------------------------------------+
+|| internalName        || Name of field for internal name. Used by the deployer           |
++----------------------+------------------------------------------------------------------+
+|| ints                || Used by the deployer                                            |
++----------------------+------------------------------------------------------------------+
+|| floats              || Used by the deployer                                            |
++----------------------+------------------------------------------------------------------+
+|| strings             || Used by the deployer                                            |
++----------------------+------------------------------------------------------------------+
+|| longs               || Used by the deployer                                            |
++----------------------+------------------------------------------------------------------+
+|| texts               || Used by the deployer                                            |
++----------------------+------------------------------------------------------------------+
+|| booleans            || Used by the deployer                                            |
++----------------------+------------------------------------------------------------------+
+|| doubles             || Used by the deployer                                            |
++----------------------+------------------------------------------------------------------+
+|| date                || Used by the deployer                                            |
++----------------------+------------------------------------------------------------------+
+|| points              || Used by the deployer                                            |
++----------------------+------------------------------------------------------------------+
+|| includeKeys         || Used by the deployer                                            |
++----------------------+------------------------------------------------------------------+
+|| includeValues       || Used by the deployer                                            |
```

Comment:
These are not field names, it's an internal identifier in ES only. We can remove them from the list